app/serializers/v1/membership_serializer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module V1
+  class MembershipSerializer < ModelSerializer
+    def serialize
+      {
+        lago_id: model.id,
+        user_id: model.user_id,
```

Comment:
what about `lago_user_id` and `lago_organization_id` so that it is aligned with other serializer?